lib/matplotlib/figure.py

```diff
@@ -1201,6 +1201,61 @@ def text(self, x, y, s, fontdict=None, **kwargs):
         self.stale = True
         return text
 
+    def set_subplotparams(self, subplotparams={}):
+        """
+        Set the subplot layout parameters.
+        Accepts either a `.SubplotParams` object, from which the relevant
+        parameters are copied, or a dictionary of subplot layout parameters.
+        If a dictionary is provided, this function is a convenience wrapper for
+        `matplotlib.figure.Figure.subplots_adjust`
+        Parameters
+        ----------
```

Comment:
You need spaces here for the docstring to be correctly rendered:

```suggestion

        Parameters
        ----------

```